lib/locales/en/chiquito.yml

```
@@ -0,0 +1,63 @@
+en:
+  faker:
+    chiquito:
+      expressions: [
+        "¡Aguaaa, aguaaa!",
+        "¡Al ataqueer!",
+        "¡Ereh un torpedo!",
+        "¡Fuegorrrl!",
+        "¡Hasta luego Lucas!",
+        "¡Hombre malo, violento!",
+        "¡Me cago en tuh muelah!",
+        "¡Noorl!",
+        "¡Reláhese usted!",
+        "¿Te da cuen?",
```

Should `da` have an h at the end given that the 's' is omitted?
```suggestion
        "¿Te dah cuen?",
```